source/stylesheets/fonts.css.scss

```diff
@@ -5,3 +5,4 @@
 @import url(https://fonts.googleapis.com/css?family=Raleway);
 @import url(https://fonts.googleapis.com/css?family=Eagle+Lake);
 @import url(https://fonts.googleapis.com/icon?family=Material+Icons);
+@import url(https://fonts.googleapis.com/css?family=Philosopher);
```

Comment:
Let's strongly consider not adding any more fonts if we don't have to. It would be better if this font is only for the Savadoro title that that be converted to an SVG path. Each font requires up to 5 async network calls.